### PR TITLE
Converting remaining akka.io links to HTTPS (#1620)

### PR DESCRIPTION
**Pull request by bpg**

### CODE REVIEW

**Comment by jonas** [comment]
Fixes #1620

File: README.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ All of our forums, chat rooms, and issue trackers are governed by our [Code Of C
 
 In addition to that, you may enjoy following:
 
-- The [news](http://akka.io/news) section of the page, which is updated whenever a new version is released
-- The [Akka Team Blog](http://blog.akka.io)
+- The [news](https://akka.io/news) section of the page, which is updated whenever a new version is released
```
Comment (jonas) [inline README.md]: It looks like this has been moved to https://akka.io/blog/news-archive.html

File: README.md [hunk 2]
```diff
@@ -40,8 +40,8 @@ All of our forums, chat rooms, and issue trackers are governed by our [Code Of C
...
+- The [Akka Team Blog](https://blog.akka.io)
```
Comment (jonas) [inline README.md]: ... and this redirects to https://akka.io/blog/

Comment (bpg) [reply README.md]: ok, let me fix that

**Pull request merged**